cea/optimization/preprocessing/decentralized_buildings.py

```diff
@@ -288,10 +288,11 @@ def calc_new_load(mdot, TsupDH, Tret, gv):
         dico[ "EforGHP" ] = resourcesRes[:,2]
         dico[ "QfromGHP" ] = resourcesRes[:,3]
 
-        os.chdir(locator.get_optimization_disconnected_folder())
-        fName = building_name + "_result.csv"
+        # os.chdir(locator.get_optimization_disconnected_folder())
+        # fName = building_name + "_result.csv"
         results_to_csv = pd.DataFrame(dico)
-        fName_result = "DiscOp_" + fName[0:(len(fName)-4)] + ".csv"
+        # fName_result = "DiscOp_" + fName[0:(len(fName)-4)] + ".csv"
```

Comment:
you can also use `os.path.splitext(fName)[0]`. Or, if you want to be lazy, `fName[0:-4]`. oops, that's not your code. you just commented it out. my bad. delete this whole line.